rxandroid/src/main/java/rx/android/view/OnSubscribeViewClick.java

```diff
@@ -75,7 +75,7 @@ public boolean removeOnClickListener(final View.OnClickListener listener) {
 
         @Override
         public void onClick(final View view) {
-            for (final View.OnClickListener listener : listeners) {
+            for (final View.OnClickListener listener: new ArrayList<View.OnClickListener>(listeners)) {
```

Comment:
Thanks for the reasonable comment. 
I modified the codes to match it, and pushed them.